tests/integration/test_genesys.py

```diff
@@ -202,6 +215,7 @@ def test_download_reports(mock_download_files, var_dictionary):
     g = Genesys()
     g.ids_mapping = var_dictionary["ids_mapping"]
```

Comment:
You are right, I could also use the get method to "get" the value, but I normally use it if I don't know whether the value is in the dictionary or not, because then, I can give a default value. If I already know the exact name of the value, I simply use ["var_name"].
I will take this into consideration for future pytest files. Thanks Michal!